packages/next/build/polyfills/process.js

```diff
@@ -0,0 +1,5 @@
+if (global.process) {
+  module.exports = global.process
+} else {
+  module.exports = require('../../compiled/process')
+}
```

Comment:
```suggestion
module.exports = global.process || require('../../compiled/process')
```

You could shorten that a bit, but that's not that important